skllm/llm/gpt/mixin.py

```diff
@@ -212,7 +218,7 @@ def _get_embeddings(self, text: np.ndarray) -> List[List[float]]:
 
 # for now this works only with OpenAI
 class GPTTunableMixin(BaseTunableMixin):
-    _supported_tunable_models = ["gpt-3.5-turbo-0125", "gpt-3.5-turbo"]
+    _supported_tunable_models = ["gpt-3.5-turbo-0613", "gpt-3.5-turbo"]
```

Comment:
Is there a reason for this change?

Reply:
that was a mistake, I'm not sure how that got in.